### PR TITLE
Add proxy-url example to kubectl documentation

### PR DESCRIPTION
**Pull request by RoSk0**
I was pointed to https://github.com/kubernetes/client-go/issues/351#issuecomment-626752837 but wasn't able to find any mention of `proxy-url` in docs.

### CODE REVIEW

File: content/en/docs/concepts/configuration/organize-cluster-access-kubeconfig.md [hunk 1]
```diff
@@ -148,7 +148,28 @@ File references on the command line are relative to the current working director
...
 
+## Proxy
 
+You can configure `kubectl` to use proxy by setting `proxy-url` in the kubeconfig file, like:
+
+```yaml
+apiVersion: v1
+kind: Config
+
+proxy-url: https://proxy.host/3128
```
Comment (jihoon-seo) [inline content/en/docs/concepts/configuration/organize-cluster-access-kubeconfig.md:159]: nit 😊

```suggestion
proxy-url: https://proxy.host:3128
```

**Review by jihoon-seo — COMMENTED** [review]
@RoSk0 PTAL!

**Comment by RoSk0** [comment]
> @RoSk0 PTAL!

HI https://github.com/jihoon-seo , I would appreciate meaningful message instead of the set of characters.

**Comment by reylejano** [comment]
Preview of updated page:
https://deploy-preview-32245--kubernetes-io-main-staging.netlify.app/docs/concepts/configuration/organize-cluster-access-kubeconfig/

/lgtm

**Pull request merged**